br/pkg/restore/snap_client/pipeline_items.go

```diff
@@ -390,112 +386,6 @@ func concurrentHandleTablesCh(
 	}
 }
 
-// GoCreateTables create tables, and generate their information.
-// this function will use workers as the same number of sessionPool,
-// leave sessionPool nil to send DDLs sequential.
-func (rc *SnapClient) GoCreateTables(
```

Comment:
move this function to `br/pkg/restore/snap_client/client.go: CreateTables`.

Maybe we can create a new file called `create_tables.go` to save the client functions and wrap the part of steps from `br/pkg/task/restore.go: runRestore`.